src/net/util/show_packet.h

```diff
@@ -20,7 +20,7 @@ static inline void show_packet(uint8_t *raw, int size, char *title) {
 	if (!&mod_logger)
 		return;
 
-	if (&mod_logger.logging &&
+	if (//&mod_logger.logging &&
```

Comment:
My concerns about this line https://github.com/embox/embox/pull/1024#discussion_r95756876 are still there. I'd prefer remove this and add assert in logging_raw